LABSN_Windows_Setup.txt

```diff
@@ -0,0 +1,93 @@
+# This is a list of commands to be run when setting up a Windows system for
+# psychoacoustics at LABS^N. It is not much of a "script" in that it is mostly
+# instructions on what to download and double-click, and in what order.
+# However, there are a few lines of code runnable in Git Bash that will save
+# you some setup time. It is recommended that you proceed in serial order
+# (don't skip around).
+
+# MATLAB
+# this has to be installed by I-LABS I.T. staff (for licensing reasons). It is
+# recommended to do this first, in case anything goes wrong and the machine
+# needs to be wiped.
+
+# PSYCHTOOLBOX
+# Prerequisites: install SVN, GStreamer, & "MS Visual C++ 2010 redistributable"
+# from downloadable EXEs (these links are current as of 2015 December):
+# https://sliksvn.com/download/
+# http://gstreamer.freedesktop.org/
+# https://www.microsoft.com/en-us/download/details.aspx?id=14632
+# After that, follow the instructions at http://psychtoolbox.org/
+
+# MISCELLANEOUS PROGRAMS: install from downloadable EXEs
+# 7zip: http://www.7-zip.org/
+# VirtualCloneDrive: http://www.slysoft.com/en/virtual-clonedrive.html
+# Git Bash: https://git-scm.com/downloads
+# Miniconda (python 2.7, 64-bit): http://conda.pydata.org/miniconda.html

```

Comment:
will add that, but surely you mean installing from EXE, not from Ubuntu Repo?  This is for Windows machines.